content/log/explore.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Explore
+kind: Documentation
+autotocdepth: 2
+hideguides: true
+customnav: lognav
+---
+<div class="alert alert-info">
+Datadog's log management solution is actualy in private beta. If you'd like to apply to participate in the private beta, please fill out <a href="https://www.datadoghq.com/log-management/">this form</a>.
+</div>
+
+## Overview 
+
+The Log explorer is an optimized view which makes it easy to do any kind of troubleshooting and data exploration.
+
+{{< img src="log/explore/explore_view_with_comments.png" alt="Explore view with comments" >}}
+
+On this view you have:
+
+* [Time range](#time-range)
+* [Log list](#log-list)
+* [Columns](#columns)
+* [Facets](#facets)
+* [Search Bar](#search-bar)
+
+## Time Range
+Time range allows you to display logs on a given time period. It is symbolised by the timeline directly under the search bar. The timeline can be displayed or wrapped up with the **Show Graph** check box:
+
+{{< img src="log/explore/timeline.png" alt="Timeline" >}}
+
+Change the time periode displayed with quick ranges that are directly available:
+
+{{< img src="log/explore/timerange.png" alt="Timerange" style="width:75%;">}}
+
+## Log list
+The log list is the list of events that match the selected context, a context is defined by a [search bar](#search-bar) filter and a [time range](#time-range).
+{{< img src="log/explore/log_list.png" alt="Log List" >}}
+
+### Sorting the log list
+
+Click on the **date** columns to get your logs in an ascending or descending order in the Log list.
+
+### Filtering the log list
+If you enter a valid query into the [search bar](#search-bar),  words that match your query are highlighted and log displayed match your facet criterias:
+
+Searching for all the logs containing the word "deleted" from host:i-0223afcd9cb2fa6a2 : 
+{{< img src="log/explore/log_list_highlighted.png" alt="Log List highlighted" >}}
+
+### Displaying a full log
+To display detail of a log line, just click on it:
```

Comment:
We need more details here about:

- What are the different sections: Severity, date, host, service, tags and log content
- Just probably mention and illustrate actions over any attribute

Reply:
We will do it on the screenshot directly with a nice overlay